force-app/main/default/classes/InterviewEventTriggerTest.cls

```diff
@@ -1,12 +1,65 @@
 @isTest
 public class InterviewEventTriggerTest {
-    @isTest static void InterviewEventTriggerInsert() {
 
-        Interview__c interview1 = new Interview__c(Name='Movie 1', InterviewStartDate__c= date.today(), Duration__c= 1);
+    @TestSetup
+    static void setup(){
+         TestDataFactory.defaultValueProvider = new MyDefaultValueProvider();
 
-        insert interview1;
+        Candidate__c newCanditate = (Candidate__c)TestDataFactory.createSObject('Candidate__c', new Map<String,Object>{
+        'First_Name__c' => 'Test',
+        'Last_Name__c' => 'Candidate'
```

Comment:
You can have "Test" and "Candidate" as constants (final static) that you can declare in this class or in a separate constants class Constants.cls